mep/people/management/commands/export_locations.py

```diff
@@ -0,0 +1,119 @@
+"""
+Manage command to export member data for use by others.
+
+Generates a CSV and JSON file including details on every library member in the
+database, with summary details and coordinates for associated addresses.
+
+"""
+
+from collections import OrderedDict
+from django.db.models import Prefetch
+from mep.common.management.export import BaseExport
+from mep.common.templatetags.mep_tags import domain
+from mep.common.utils import absolutize_url
+from mep.people.models import Location, Person
+from mep.accounts.models import Account, Address
+from mep.accounts.partial_date import DatePrecisionField
+
+
+class Command(BaseExport):
+    """Export member data."""
+
+    help = __doc__
+
+    model = Address
+
+    csv_fields = [
+        "id",  # including "id" to store slug for exports,
+        # given not all exported entities have a URI
+        "member_id",  # member slug
+        "member_uri",
+        "care_of_person_id",  # person slug
+        "street_address",
+        "postal_code",
+        "city",
+        "arrrondissement",
+        "country",
+        "start_date",
+        "end_date",
+        "start_date_precision",
+        "end_date_precision",
```

Comment:
please access and export using the partial date descriptor, which should handle all the logic for you - I think it should be `partial_start_date` and `partial_end_date` (this is probably not documented as clearly as it could be); 
don't export the precision separately